.github/workflows/build.yml

```
@@ -29,9 +29,15 @@ jobs:
           ./mvnw clean install
       
       - name: Upload coverage reports to Codecov
+        if: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name }}
         uses: codecov/codecov-action@v3
         with:
           fail_ci_if_error: true
+          token: ${{ secrets.CODECOV_TOKEN }}
+      
+      - name: Upload coverage reports to Codecov from fork
```

```suggestion
      - name: Upload coverage reports to Codecov
```
can we use the same name?